Add community information and fix broken links (#382)
kserve/website: docs/community/get_involved.md

# How to Get Involved

Welcome to the KServe community!

> Ask questions, join the conversation, or contribute to KServe's development. As an open-source project, KServe thrives on the active participation of its users and contributors. Let's work together to make machine learning model serving effortless. Join us!

Reviewer: ```suggestion
Feel free to ask questions, engage in discussions, or get involved in the KServe's development. KServe, as an open-source project, thrives on the active participation of its community. Let's work together to make machine learning model serving effortless. Join us!
```

## How do you want to get involved?

### Ask Questions

> For the fastest response, you can ask questions on the`#kserve` channel of the [CNCF

Reviewer: ```suggestion
For the fastest response, you can ask questions on the `#kserve` channel of the [CNCF
```

Slack](https://slack.cncf.io/).

> To Join the channel, [Create your CNCF Slack account](https://slack.cncf.io/) and Search for the `#kserve` channel or join via [this link](https://cloud-native.slack.com/archives/C06AH2C3K8B).

Reviewer: ```suggestion
To Join the channel, [create your CNCF Slack account](https://slack.cncf.io/) and search for the `#kserve` channel or join via [this link](https://cloud-native.slack.com/archives/C06AH2C3K8B).
```

> If you prefer, GitHub discussions, you can join the [KServe discussions](https://github.com/kserve/kserve/discussions).

Reviewer: ```suggestion
If you prefer to use GitHub discussions, you can join the [KServe discussions](https://github.com/kserve/kserve/discussions).
```

### Bug Reports and Feature Requests

We use GitHub Issues to track bug reports and feature requests. Please file your issues and feature requests in the [KServe main repository](https://github.com/kserve/kserve/issues/new/choose).

For Documentation related issues, please use the [KServe Website repository](https://github.com/kserve/website/issues/new/choose).

For Open Inference Protocol (V2) related issues and feature requests, please use [Open Inference Protocol repository](https://github.com/kserve/open-inference-protocol/issues/new)

A good bug report should include:

> - What you were trying to do, and what happened

Reviewer: ```suggestion
- Description: Clearly state what you were trying to accomplish and what behavior you observed instead
```

> - What version of KServe, Knative (If Serverless), Kubeflow (If used with Kubeflow) and Kubernetes you are using (if using a cloud provider, indicate which one)

Reviewer: ```suggestion
- Versions: Specify the versions of relevant components
   - KServe version
   - Knative version (If using Serverless)
   - Kubeflow version (If used with Kubeflow)
   - Kubernetes version 
   - Cloud provider details (if using a cloud provider, indicate which one)
```

> - Relevant resource yaml, HTTP requests, or log lines

Reviewer: ```suggestion
- Relevant Artifacts:
   - Resource YAML files (InferenceService, InferenceGraph)
   - HTTP request details
   - Relevant log lines or error messages
```
